src/Components/WebAssembly/testassets/StandaloneApp/_Imports.razor

```diff
@@ -2,5 +2,6 @@
 @using System.Net.Http.Json
 @using Microsoft.AspNetCore.Components.Routing
 @using Microsoft.AspNetCore.Components.Web
+@using Microsoft.AspNetCore.Components.WebAssembly.Http
```

Comment:
If this is also going into the project template then good, or if not then please revert.